index.html

```diff
@@ -49,7 +52,7 @@
 
 
     <main class="main-index">
-      <img class="logo-main" src="#" alt="Добро пожаловать в прекрасную Седону. «Потому что Гранд Каньон отстой!»">
+      <img class="logo-main" src="images/logo/welcome.png" width="458" height="352" alt="Добро пожаловать в прекрасную Седону. «Потому что Гранд Каньон отстой!»">
       <h1 class="visually-hidden">Седона</h1>
```

Comment:
Лучше и надежнее задавать фон не картинке а обернуть его в какой-то блок
```suggestion
      <header class="logo-main">
        <img src="images/logo/welcome.png" width="458" height="352" alt="Добро пожаловать в прекрасную Седону. «Потому что Гранд Каньон отстой!»">
        <h1 class="visually-hidden">Седона</h1>
      </header>
```